src/domains/product/product-not-found.js

```diff
@@ -0,0 +1,6 @@
+export default class OutOfStockError extends Error {
```

Comment:
plutôt `ProductNotFoundError` non ?

Reply:
Non ProductNotFound ce serait plus si tu cherchais à obtenir un produit qui n'existe juste pas.

Genre acheter un sextoy alors que le magasin en vend pas, il est pas en rupture de stock, juste ça n'existe pas ( et dans ce cas oui 404 not found)

Reply:
yes ça j'ai compris, par contre ce que je vois c'est le nom de ton fichier VS le nom de ta classe ;)

Reply:
oui je m'en suis rendu compte j'ai fix aussi